sklearn/metrics/tests/test_ranking.py

```diff
@@ -426,7 +427,20 @@ def test_auc_errors():
     assert_raises(ValueError, auc, [0.0], [0.1])
 
     # x is not in order
-    assert_raises(ValueError, auc, [1.0, 0.0, 0.5], [0.0, 0.0, 0.0])
+    error_message = ("x is neither increasing nor decreasing : "
+                     "[ 2.5  1.6  3.7  4.8]. np.diff(x) contains 2 positive "
```

Comment:
numpy 1.14 will change np array repr/str so don't hard-code it like this instead use `'{}'.format(your_array)`.